lib/command/maintenance_on.rb

```diff
@@ -48,7 +48,7 @@ def call # rubocop:disable Metrics/MethodLength
         cp.set_domain_workload(domain_data, maintenance_workload)
 
         # Give it a bit of time for the domain to update
-        sleep 30
+        Kernel.sleep(30)
```

Comment:
@dzirtusss we should find a better way to poll for success than a blunt wait of 30 seconds. Let's add an "issue" for this.

Reply:
https://github.com/shakacode/heroku-to-control-plane/issues/157